tests/naive.py

```diff
@@ -156,7 +157,7 @@ def stamp(T_A, m, T_B=None, exclusion_zone=None):
     return result
 
 
-def stump(T_A, m, T_B=None, exclusion_zone=None):
+def stump(T_A, m, T_B=None, exclusion_zone=None, row_wise=False):
     """
     Traverse distance matrix along the diagonals and update the matrix profile and
     matrix profile indices
```

Comment:
Can you please add an additional note that when `row_wise=True` then what happens?

Reply:
Sure. I will add note in this doc string.